sklearn/decomposition/dict_learning.py

```diff
@@ -669,8 +670,9 @@ def dict_learning_online(X, n_components=2, alpha=1, n_iter=100,
                 print ("Iteration % 3i (elapsed time: % 3is, % 4.1fmn)"
                        % (ii, dt, dt / 60))
 
+        # Setting n_jobs = 1 to avoid creating workers at each iteration
         this_code = sparse_encode(this_X, dictionary.T, algorithm=method,
-                                  alpha=alpha, n_jobs=n_jobs).T
+                                  alpha=alpha, n_jobs=1).T
```

Comment:
are you making n_jobs param useless? I am surprised there is no speed gain
with n_jobs. It's not what I remember from benching this.